source/library/Monadoc/Action/CronEntry/Enqueue.hs

```diff
@@ -44,19 +44,19 @@ runOne cronEntry = do
   pure (job, newCronEntry)
 
 enqueueJob :: CronEntry.Model -> App.App Job.Model
-enqueueJob = Job.Enqueue.run . CronEntry.task . Model.value
+enqueueJob = Job.Enqueue.run . (.task) . (.value)
 
 updateRunAt :: CronEntry.Model -> App.App CronEntry.Model
 updateRunAt cronEntry = do
   now <- Timestamp.getCurrentTime
   runAt <- nextRunAt now cronEntry
-  let newCronEntry = cronEntry {Model.value = (Model.value cronEntry) {CronEntry.runAt = runAt}}
+  let newCronEntry = cronEntry {Model.value = cronEntry.value {CronEntry.runAt = runAt}}
```

Comment:
It's nice that the precedence works out for stuff like this. Lots fewer parentheses!